easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.2.1-CrayGNU-2016.11-cuda-8.0-Python-3.5.2.eb

```diff
@@ -0,0 +1,68 @@
+# @author: marcelsc, victorusu, gppezzi
+
+easyblock = "CmdCp"
+
+name = 'TensorFlow'
+version = '1.2.1'
+pyver = '3.5.2'
+cudaver = '8.0'
```

Comment:
Sure. But for the next software stack update, we'll remove this last part from the suffix from all apps.